components/modals/FeedbackSentModal.tsx

```diff
@@ -1,15 +1,67 @@
+import { useTheme } from '@react-navigation/native';
+import { useRouter } from 'expo-router';
 import React from 'react';
-import { View } from 'react-native';
+import { Image, Modal, Pressable, StyleSheet, View } from 'react-native';
+import StyledText from '../common/StyledText';
 
-// TODO: Feedback sent modal
-/**
- * - Implement a modal that displays once you send feedback as seen in design
- * - Clicking on the Okay button should close modal
- * - closing modal should navigate to home page
- */
+const FeedbackSentModal = (props: { showModal: boolean }) => {
+  const { colors } = useTheme();
+  const router = useRouter();
 
-const FeedbackSentModal = () => {
-  return <View />;
+  const { showModal } = props;
+  return (
+    <Modal visible={showModal} transparent animationType="fade">
+      <View style={styles.modalView}>
+        <View style={[styles.container, { backgroundColor: colors.background }]}>
+          <Image style={styles.confetti} source={require('../../assets/images/confetti.gif')} />
```

Comment:
use Image from `expo-image`

Reply:
Fixed in commit 22100457842deafb9bc65ac303210c1ba2fc32c4

Reply:
did you test this? because the confetti is not showing on ios simulator or even device